modules/gnome/hm.nix

```diff
@@ -14,14 +14,22 @@ with lib;
         picture-uri = "file://${config.stylix.image}";
         picture-uri-dark = "file://${config.stylix.image}";
       };
+      
+      "org/gnome/desktop/interface" = with config.stylix.fonts ; {
+        # We show the same colours regardless of this setting, and the quick
+        # settings tile is removed. The value is still used by Epiphany to
+        # request dark mode for websites which support it.
+        color-scheme =
+          if config.stylix.polarity == "dark"
+          then "prefer-dark"
+          else "default";
 
-      # We show the same colours regardless of this setting, and the quick
-      # settings tile is removed. The value is still used by Epiphany to
-      # request dark mode for websites which support it.
-      "org/gnome/desktop/interface".color-scheme =
-        if config.stylix.polarity == "dark"
-        then "prefer-dark"
-        else "default";
+        # Some GTK apps will use these font settings if they exist.
+        # i.e emacs-pgtk.
+        font-name = "${sansSerif.name} ${toString sizes.applications}";
+        document-font-name = "${serif.name}  ${toString (sizes.applications - 1)}";
```

Comment:
Why is this 1 less than the chosen size?

Reply:
It's what GNOME does.